app/src/main/java/org/mifos/mobile/utils/Utils.kt

```diff
@@ -9,9 +10,12 @@ import android.graphics.drawable.LayerDrawable
 import android.net.Uri
 import android.util.Log
 import android.view.Menu
+import android.view.inputmethod.InputMethodManager
```

Comment:
Are you sure this import and the others which are added by you are being used?

Reply:
No they aren't, I've removed them now.